test/functional/rpc_fundrawtransaction.py

```diff
@@ -28,6 +28,9 @@ class RawTransactionsTest(BitcoinTestFramework):
     def set_test_params(self):
         self.num_nodes = 4
         self.setup_clean_chain = True
+        # This test isn't testing tx relay. Set whitelist on the peers for
+        # instant tx relay.
+        self.extra_args = [['-whitelist=127.0.0.1']] * 4
```

Comment:
Could do
```diff
     def set_test_params(self):
-        self.num_nodes = 4
+        num_nodes = 4
+        self.num_nodes = num_nodes
         self.setup_clean_chain = True
         # This test isn't testing tx relay. Set whitelist on the peers for
         # instant tx relay.
-        self.extra_args = [['-whitelist=127.0.0.1']] * 4
+        self.extra_args = [['-whitelist=127.0.0.1']] * num_nodes
```

Reply:
even better, just:

```
        self.extra_args = [['-whitelist=127.0.0.1']] * self.num_nodes
```

which I've now done.

Reply:
Yes, even better; doing the same in wallet_avoidreuse.py